Sources/MotorControlUI/OverviewView.swift

```diff
@@ -72,7 +72,7 @@ struct OverviewNodeView: View {
         GeometryReader { scrollViewGeometry in
             ScrollViewReader { proxy in
                 ScrollView {
-                    VStack(alignment: .center, spacing: spacing) {
+                    VStack(alignment: .center) {
```

Comment:
While this works... might be better to think of the image and the content as two different subviews. Then you can have all the content have a horizontal padding, spacing, ect. that avoids all the custom padding and still keeps the image as edge to edge.